unixfs/mod/dagmodifier.go

```diff
@@ -0,0 +1,428 @@
+package mod
```

Comment:
Yeah, git diff apparently doesnt recognize them as the same code anymore, or else it would have shown up as a rename.